Parallelize untar operation in case of gridpack

Qubitol opened the pull request:
This PR improves the untarring of the zip files created when using the `gridpack` option, by parallelizing the operation using `xargs`.

On a separate, but related, issue, the unzipping is performed by following lines of `bin/internal/restore_data`:
```bash
gunzip \$j
cp $1_ftn26 ftn26
gzip $1_ftn26
```
However, re-zipping the file can be avoided by using the `--keep` option of `gunzip`, maybe we can save even more time.

Qubitol committed:
Add parallelization to unzip operations

Qubitol committed:
Optimize loop to consider Process dirs alltogether

oliviermattelaer commented:
Hi Daniele,

Yes you can use the --keep this is a very good idea!.

Then this is good to merge.

Thanks a lot,

Olivier

Qubitol committed:
Keep the unzipped file to not rezip it again

Qubitol commented:
Hi Olivier,

I added the `--keep` and changed the copy to an `mv` in the line next to it.
If you agree, we can merge, but apparently I don't have write access to the repo.

Thanks.

Daniele

oliviermattelaer commented:
Thanks, I have done it myself then :-)
